caracal/workers/inspect_worker.py

```diff
@@ -239,13 +239,11 @@ def get_cfg_fields(pipeline, iobs, cfg_field, label_in):
     fields = {}
     for f_type in f_types:
         fnames = getattr(pipeline, f_type)[iobs]
-        fids = getattr(pipeline, f"{f_type}_id")[iobs]
-
-        for _fid, _fname in zip(fids, fnames):
-            # set field id to 0 if the active MS is a target only split
-            if label_in != '' and f_type == "target":
-                _fid = 0
-            fields[_fname] = (f_type, _fid)
+        for _fname in fnames:
+            if _fname in fields.keys():
+                fields[_fname].append(f_type)
+            else:
+                fields[_fname] = [f_type]
```

Comment:
OK, just one question: the old code collected field IDs (with the target field ID forced to 0 since the target is always assumed to be extracted into its own MS), the new code doesn't seem to use field IDs anymore -- have you eliminated them completely? (Probably a good thing!...)

Also, that whole if-else construct can be replaced by a single statement:

```
fields.setdefault(_fname, []).append(f_type)
```

Reply:
Tagging @Mulan-94 since I'm not sure the notification arrives otherwise...

Reply:
Yeah I removed field ids since I didn't see where they were being used.

> Also, that whole if-else construct can be replaced by a single statement:

Thank you! let me change